frontend/.env.local.template

```diff
@@ -1,5 +1,5 @@
 
-VITE_SUPABASE_URL='<URL>'
-VITE_SUPABASE_KEY='<KEY>'
-VITE_DDDICE_AUTH_KEY='<KEY>'
+VITE_SUPABASE_URL='<API_URL>'
+VITE_SUPABASE_KEY='<ANON_KEY>'
+VITE_DDDICE_AUTH_KEY='<ANON_KEY>'
```

Comment:
I believe this is a different key, so for the sake of clarity, this should be different

Reply:
Aah - that got me when I tried going through the setup. I really didn't understand where to put the anon key, but probably `VITE_DDDICE_AUTH_KEY` needs a different value.
Am I right, that this is some auth key to [dddice](https://dddice.com)? If that's the case, I probably wouldn't be able to roll dice on my locally hosted version of Wanderer's Guide, if the auth key for dddice isn't valid?
I'll have a look how to obtain an auth key from there and add that to the setup.

Reply:
Yes, you're right. You can't roll virtual dice without the API key, but the roller feature still works without it.